bindings/python/python/opendal/__init__.pyi

```diff
@@ -145,7 +146,7 @@ class Operator(_Base):
         Returns:
             An iterable of entries representing the objects in the directory.
         """
-    def scan(self, path: PathBuf) -> Iterable[Entry]:
+    def scan(self, path: PathBuf, **kwargs) -> Iterable[Entry]:
```

Comment:
Please mark this API as deprecated. It's a long wait issue that remove the `scan` concept.

Reply:
```python
from warnings import deprecated

@deprecated
def demo():
	pass
```

Reply:
BTW  @Xuanwo I think we don't need to modify the pyi file manually right?

Reply:
@Zheaoli maybe im wrong here, but wasn't this `deprecated` decorator recently added in latest [python 3.13](https://peps.python.org/pep-0702/)?

opendal supports python version back to 3.10

<img width="556" alt="image" src="https://github.com/user-attachments/assets/6208f055-e337-4d46-ab35-2f4974bce31b" />

Reply:
> BTW @Xuanwo I think we don't need to modify the pyi file manually right?

Any suggestions?

Reply:
Looks nice. Would you like to create an issue for this? I think we can give it a try.

Reply:
done